earthmover/nodes/source.py

```diff
@@ -266,13 +265,29 @@ def __get_skiprows(config: 'YamlMapping'):
             _header_rows = config.get('header_rows', 1)
             return int(_header_rows) - 1  # If header_rows = 1, skip none.
 
+        def __read_fwf(file: str, config: 'YamlMapping'):
+            colspec_file = os.path.join(os.path.dirname(self.config.__file__), config.get('colspec_file'))
```

Comment:
Need to tweak this so that it is safer in the event that no `colspec_file` is specified